docs/topics/mta-rn-known-issues-6-2-0.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-6-2-0_{context}"]
+= Known issues
+
+MTA version 6.2.0 has the following issues.
+
+.MTA fails to run analysis in private GitHub repo
+Fail to run analysis in private github repo. link:https://issues.redhat.com/browse/MTA-810[MTA-810]
+
+.Jira connection (instance) cannot be removed 
+Jira connections cannot be removed after it was added to Wave even after Wave was removed. link:https://issues.redhat.com/browse/MTA-1014[MTA-1014]  
+
+See an updated  filtered list of link:https://issues.redhat.com/issues/?filter=12418204[known issues in Jira].
```

Comment:
@HagayVider1 - here is how I would like the Known Issues to look. What do you think?

Please reach out to Dylan and ask him if there are any other known issues he wants adding? Also ask for more details about the two known issues (if he wants them keeping)

Please can you check the Jira filter and see if you can figure out what is going wrong as I cannot access it.